docs/source/ui/pages/run_management.md

```diff
@@ -67,9 +68,22 @@ Each tab visualizes respective tracked metadata or empty if not tracked.
 - [Figures](#id10)
 - [Settings](#id11)
 
+### Overview Tab
+
+Overview tab shows overall info about the run.
+- These cards can contain information about `Parameters`, `Metrics`, `System Metrics`, `CLI Arguments`, `Environment Variables`, `Packages` and `Git information`. With this data, you can easily reproduce your run.
+
+- Sidebar contains information about  `Run Date`, `Run Duration`, `Run Hash`, attached `Tags` and gives the ability to navigate through tabs.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Single Run page params tab" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/run-single-page/overview_tab.png">
+
+Also, you can do searching in table, and it will return value by `Match Case`, `Match Word` and `Match Regexp`
```

Comment:
Maybe would be better to say:
Also, you can apply advanced searching/filtering to those card tables.